common4j/src/main/com/microsoft/identity/common/java/cache/CacheKeyValueDelegate.java

```diff
@@ -95,22 +94,16 @@ public CacheKeyValueDelegate() {
         static final String REQUESTED_CLAIMS = "<requested_claims>";
     }
 
-    private static String sanitizeNull(final String input) {
-        String outValue = null == input ? "" : input.toLowerCase(Locale.US).trim();
-
-        return outValue;
-    }
-
     @Override
     public String generateCacheKey(AccountRecord account) {
         String cacheKey = HOME_ACCOUNT_ID
                 + CACHE_VALUE_SEPARATOR
                 + ENVIRONMENT
                 + CACHE_VALUE_SEPARATOR
                 + REALM;
-        cacheKey = cacheKey.replace(HOME_ACCOUNT_ID, sanitizeNull(account.getHomeAccountId()));
-        cacheKey = cacheKey.replace(ENVIRONMENT, sanitizeNull(account.getEnvironment()));
-        cacheKey = cacheKey.replace(REALM, sanitizeNull(account.getRealm()));
+        cacheKey = cacheKey.replace(HOME_ACCOUNT_ID, StringUtil.sanitizeNullAndLowercaseAndTrim(account.getHomeAccountId()));
```

Comment:
intentional behavior change?

Reply:
It isn't a behavior change, it's a correct naming:  This is what sanitizeNull does.

```
    private static String sanitizeNull(final String input) {
         String outValue = null == input ? "" : input.toLowerCase(Locale.US).trim();

         return outValue;
     }
```